Chat/assignment-chat/server-chat/src/main/java/ru/assignment/model/ChatModel.java

```diff
@@ -0,0 +1,60 @@
+package ru.assignment.model;
+
+import ru.assignment.ChatMessage;
+import ru.assignment.net.Session;
+
+import java.util.*;
+
+/**
+ * Created by Андрей on 18.02.2015.
+ */
+public class ChatModel {
+    private Map<Integer, Session> listenerMap;
```

Comment:
List<ChatModelListener> listeners:

Session implements ChatModelListener;

for (ChatModelListener listener : listeners) {
    listener.notifyNewMessageMessage(message);
}